Resources/doc/reference/architecture.rst

```diff
@@ -325,7 +325,7 @@ To do this, you first need to call the ``addChild`` method in your PostAdmin ser
     .. code-block:: xml
 
         <!-- app/config/config.xml -->
-        <service id="sonata.news.admin.post" class="Sonata\NewsBundle\Admin\PostAdmin">
+        <service id="sonata.news.admin.post" class="AppBundle\Admin\PostAdmin">
```

Comment:
So I should update the name spaces below in the code examples too? I was just trying to get them all consistent. What should they be? This AppBundle\Admin is consistent with SF3 and the getting started chapter.